packages/crypto/src/__tests__/generate-key-fixture.test.ts

```diff
@@ -26,9 +26,9 @@ it('can generate test fixture', async () => {
 
   for (let i = 0; i < mnemonic.mnemonic.length; i++) {
     const m0 = mnemonic.mnemonic[i];
-    const k0 = await toKeyPair(m0, 0, 'Ed25519');
-    const k1 = await toKeyPair(m0, 0, 'X25519');
-    const k2 = await toKeyPair(m0, 0, 'secp256k1');
+    const k0 = await toKeyPair(m0, 'Ed25519', "m/44'/1'/0'/0/0");
```

Comment:
much better / safer... just beware of `/1` vs `/0` (BTC) vs `/60` (ETH)

Reply:
Copy that!